app/questions.py

```diff
@@ -1,28 +1,56 @@
-import sys
-from sympy import symbols, latex, Eq, Rational
+from sympy import symbols, latex, Eq
 
-QUESTIONS = {
-    'mcat': set(
-        '1',
-    ),
-}
+class QuestionError:
```

Comment:
This could inherit from the Exception/Error class I think

Reply:
Agree